app/pages/health/live/getLiveStatus.js

```diff
@@ -0,0 +1,5 @@
+import { status } from '../status';
+
+export async function getLiveStatus() {
```

Comment:
what is this 'function' word you have here??

Reply:
I must admit I didn't pay attention to this. I've copied it from PB. Will update to es6 style

Reply:
I would say get rid of this file completely - this function just returns a static value